sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ConnectionPolicy.java

```diff
@@ -498,4 +498,10 @@ public String toString() {
             ", maxRequestsPerConnection=" + maxRequestsPerConnection +
             '}';
     }
+
+    public String asDiagnostics() {
+        // TODO: cache this to avoid recomputation per diagnostics request
+        // TODO: only include essential configs
```

Comment:
Will it be addressed now?

Reply:
yes. once we close on the high level of the PR what information we want to capture, etc. I will address this within the scope of this PR.

Reply:
done.